Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:stretch as builder
+
+RUN mkdir -p $GOPATH/pkg/mod $GOPATH/bin
+
+RUN apt-get update \
+  && apt-get install -y ruby ruby-dev curl  \
+  && gem install --no-document fpm \
+  && curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh \
+  && mkdir -p $GOPATH/src/github.com/davidnewhall
+
+COPY . $GOPATH/src/github.com/davidnewhall/unifi-poller
+WORKDIR $GOPATH/src/github.com/davidnewhall/unifi-poller
+
+RUN dep ensure \
+  && make dockerbin
+
+FROM scratch 
+
+COPY --from=builder /go/src/github.com/davidnewhall/unifi-poller/unifi-poller.dockerbin /unifi-poller
```

Comment:
Forgive me for being a docker noob, but is it normal to put your binary at `/` ?

Reply:
yeah on this case it is normal because it is build from scratch and there are no other binaries available - you do not have e.g. a shell

Reply:
I've done a bit of "looking around" and I'm beginning to understand this a bit better. Thanks for the details!